tailwindcss.worker.d.ts

```diff
@@ -10,7 +10,7 @@ export interface TailwindWorkerOptions {
    * @returns
    *   A valid Tailwind configuration.
    */
-  prepareTailwindConfig?: (tailwindConfig?: TailwindConfig | string) => Config | PromiseLike<Config>
+  prepareTailwindConfig?: (tailwindConfig?: TailwindConfig | string) => Config
```

Comment:
Why was this changed?

Reply:
I did that in early experiment, trying to get things working. I've put it back as original afterwards, I'll fix it in next commit.